VirtualClosetApp/www/templates/closet/newItem.html

```diff
@@ -1,31 +1,45 @@
-<ion-view view-title="newItem">
+<ion-view view-title="newItem" ng-controller="newitemCtrl">
   <ion-content>
     <style>h3 {
       height: 110px;
     }
     </style>
     <h3 align="center">Add new item</h3>
 
+    <img ng-show="imgURI !== undefined" ng-src="{{imgURI}}" id = "itemImg" ng-model="itemImg">
+    <img ng-show="imgURI === undefined" ng-src="http://placehold.it/250x250">
```

Comment:
Instead of linking to an outside source for images, we should create an assets folder and call from there.